packages/rnpm-plugin-windows/src/windows.js

```diff
@@ -70,9 +70,12 @@ module.exports = async function (config, args, options) {
     const name = args[0] || Common.getReactNativeAppName();
     const ns = options.namespace || name;
     const version = options.windowsVersion || Common.getReactNativeVersion();
-    const versionTag = options.template || await getDefaultVersionTag(version);
+    let rnwPackage = version;
+    if (!version.startsWith("file:")) {
```

Comment:
>!version.startsWith("file:")) [](start = 8, length = 29)

curious, in what scenario would you pass file:xyz ? maybe add a comment?

Reply:
file: is a standard package.json versioning feature.  It says the version is not a semver, but instead a file path.  This is useful when testing local changes that haven't been published yet. 
 There are some examples here:  https://docs.npmjs.com/files/package.json